datameta/security/__init__.py

```diff
@@ -223,10 +225,13 @@ def revalidate_user(request):
         )).one_or_none()
     # Check if the user still exists and their group hasn't changed
     if user is None or user.group_id != request.session['user_gid']:
+        if user.group_id != request.session['user_gid']:
+            register_failed_login_attempt(request.dbsession, user)
```

Comment:
There is nothing wrong with this from a security / user point of view. It just means that the group association was changed by an admin while a user session was active and that the user has to re-authenticate because of that. No need to track.

Reply:
Ok, I will remove that check, but did you comment on an earlier version of the code here?